test/unit/examples/groupProvisioningRequests/multipleGroupsCreation.json

```diff
@@ -18,7 +18,7 @@
           "type": "Lumens"
         }
       ],
-      "active": [
+      "attributes": [
```

Comment:
According to issue #588 description, I understand the fixes in documentation (.md) and json in examples/. However, PR includes also this fix in unit test.

Does this means that the field that is being change is not actually used in this unit test?

(Not very important... just to be sure I'm fully understaning the PR :)

Reply:
You're right. This specific request is used in https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/mongodb/mongodb-group-registry-test.js#L297.

Nevertheless, as you can see in the following lines, the test does not include any assertion to check that attributes are present. Anyway, they would be empty! ;)

Reply:
NTC